tutorials/performance/using_multiple_threads.rst

```diff
@@ -47,6 +47,93 @@ To create a thread, use the following code:
     func _exit_tree():
         thread.wait_to_finish()
 
+ .. code-tab:: cpp C++ .H File
+
+    #ifndef MULTITHREADING_DEMO_H
+    #define MULTITHREADING_DEMO_H
+    
+    #include <godot_cpp/classes/node.hpp>
+    #include <godot_cpp/classes/thread.hpp>
+    
+    namespace godot {
```

Comment:
In the demo projects we don't put the classes in the Godot namespace, but the other extension example does, I think it's fine here like this for consistency, and we can change all the examples later to match